docs/getting-started/linux-installation.md

```diff
@@ -24,68 +24,48 @@ how to run Outrigger projects on Linux.
-1. [Install Docker Compose](https://docs.docker.com/compose/install/)
-
-**If using Upstart**
-
-- Set the DNS configuration for dnsdock, as well as known RFC-1918 address space
-    - Please note that the following command will over-write your existing Docker daemon configuration file.  Please 
-    set the -dns=172.17.0.1 option manually as an alternative
-    - `echo 'DOCKER_OPTS="-dns=172.17.0.1"' | sudo tee /etc/default/docker`
-- Start the docker daemon
-    - `sudo start docker`
-
-**If using Systemd**
-
-- Set the DNS configuration for Docker
+2. [Install Docker Compose](https://docs.docker.com/compose/install/)
+3. Set the DNS configuration for Docker
     - We need to modify the command that docker uses within systemd
     - `sudo mkdir /etc/systemd/system/docker.service.d`
     - `sudo vi /etc/systemd/system/docker.service.d/docker.conf`
-    - In that file put something like the following:    
-    
+    - In that file put something like the following:      
 ```bash
 [Service]
 ExecStart=
 ExecStart=/usr/bin/dockerd -H fd:// --dns=172.17.0.1
 ```
+4. Restart the docker daemon
+    - `sudo systemctl restart docker`
 
-## Linux DNS configuration options
+## Automated Linux DNS configuration options
 
-### Method 1: dnsmasq via NetworkManager
+Outrigger will help automate the setup of DNS (via `rig start` and `rig dns`) if you are using 
+one of the following options for name resolution:
 
-This method works well with no other needed software provided that you have unfettered access to your system's 
-configuration, and are using NetworkManager to maintain your networking stack
+1. dnsmasq via NetworkManager
+2. libnss-resolver
 
-1. Add the line dns=dnsmasq to /etc/NetworkManager/NetworkManager.conf under the [main] configuration stanza. This will 
-cause NetworkManager to spawn and use a dnsmasq process for all name resolution. If you already have a local configuration, 
-ensure that it is not configured to start on system boot.
-1. Add a single rule to direct all DNS lookups for .vm addresses to the 172.17.0.1 address.
-    - `echo 'server=/vm/172.17.0.1' | sudo tee /etc/NetworkManager/dnsmasq.d/dnsdock.conf`
-1. Restart NetworkManager, either through systemd, or by simply rebooting.  To restart via systemd:
-    - `systemctl restart NetworkManager`
-1. Run the dnsdock container
+## Other DNS resolution options
 
-```bash
-docker run -d \
-  --name=dnsdock \
-  --restart=always \
-  -l com.dnsdock.name=dnsdock \
-  -l com.dnsdock.image=outrigger \
-  -p 172.17.0.1:53:53/udp \
-  -v /var/run/docker.sock:/var/run/docker.sock \
-  aacebedo/dnsdock:v1.16.1-amd64 --domain=vm
-```
+DNSDock is the service used for Outrigger service DNS resolution. If you use different services to 
+manage DNS resolution in your operating system, then you will need to configure it to use `172.17.0.1`
+or whatever the Docker bridge network gateway IP address to resolve for `.vm` addresses (or just generally
```

Comment:
This reads a little awkwardly. I think it's missing an is (or whatever the Docker bridge network gateway IP address is ...).

That sentence is a little complex though so perhaps swap to:

If you use different services to manage DNS resolution then you will need to configure the Docker bridge network gateway IP address to resolve `.vm` addresses. This is usually `172.17.0.1`. You can also more generally lookup names there if none of the other resolvers find a match. Alternatively . . .